### ISSUE TITLE
Improve screen reader support for  comment reactions 

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#168965]
Microsoft/vscode#69321

Tested using VoiceOver on MacOS

**Issues**
- The add reaction button on the right side is currently read out at `Menu (8 items)`

- When focusing on a reaction in the context menu, it is read as just the emoji name. Github instead reads each option as `React with smily face`

- The existing reactions do not have helpful voice text. They should be read as the reaction name and number of reactions

- The add reaction button the right of existing reactions also does not have helpful voice over labels

**Comment by alexr00** [comment]
> The add reaction button on the right side is currently read out at Menu (8 items)

This is now read as "Toggle Reaction"

> When focusing on a reaction in the context menu, it is read as just the emoji name. Github instead reads each option as React with smily face

Since this uses a context menu, this is intentional. Menus don't have special aria labels.

> The existing reactions do not have helpful voice text. They should be read as the reaction name and number of reactions

This now reads a more useful "Toggle reaction, x reactions with y" message.

> The add reaction button the right of existing reactions also does not have helpful voice over labels

Same as above since this is a menu.

**Comment by roblourens** [comment]
This seems improved but I can't get it to read the menu when I tab from the comment to the react menu button. It doesn't read anything when I use the arrow keys to move focus between menu buttons. If I expand the menu with space, then it reads the menu name. It looks like this, so I guess voiceover is stuck on the full comment context even though focus has moved. Not a voiceover expert though, is that an issue or should I be using other nav keys anyway?

<img width="610" alt="image" src="https://user-images.githubusercontent.com/323878/214924395-dca175d8-bb40-4b58-aa4f-15404c618cb6.png">
